tools/rex-info.c

```diff
@@ -74,6 +74,20 @@ void rex_dump_lineset_block (struct rex_lineset *ls)
     /*     printf ("%f %f %f\n", ls->vertices[i], ls->vertices[i + 1], ls->vertices[i + 2]); */
 }
 
+void rex_dump_pointlist_block (struct rex_pointlist *p)
+{
+    if (!p)
+        return;
+
+    printf ("nr_positions           %20d\n", p->nr_vertices);
+    printf ("nr_colors              %20d\n", p->nr_colors);
+
+    /* for (int i = 0; i < p->nr_vertices * 3; i += 3) */
```

Comment:
still neded?

Reply:
yes for debugging ;-) but maybe an ifdef would be nicer here

Reply:
i see :)